src/stravalib/client.py

```diff
@@ -1369,49 +1372,62 @@ def get_segment_efforts(
         """Gets all efforts on a particular segment sorted by start_date_local
 
         Returns an array of segment effort summary representations sorted by
-        start_date_local ascending or by elapsed_time if an athlete_id is
+        `start_date_local` ascending or by elapsed_time if an athlete_id is
         provided.
 
-        If no filtering parameters is provided all efforts for the segment
+        NOTES:
+        * The `athlete_id` sorting only works if you do NOT provide start and end dates.
```

Comment:
This is undocumented but seems to work if you don't provide a start and end date. So it's a bit of a weird feature.